### DISCUSSION TITLE
Summary Row/Update column value based on other columns

### DISCUSSION THREAD

**Discussion by fhyami** [komarovalexander/ka-table#331] [Q&A] [answered]
Hello,

I am trying to summarise rows. So if I have a row with data: price: 100 , vat: 20. I want to create a column that displays both of these values together. Is this possible, to either get this as a summary on the row, or have a column that is a sum of these two values.

Thank you, and I apologise for the bad example

**Comment by komarovalexander** [comment, marked as answer]
Hi @fhyami  heck custom cell demo https://komarovalexander.github.io/ka-table/#/custom-cell
props of the cell has props.rowData - you can use it to create a sum of columns 

I also created an example for you: https://stackblitz.com/edit/table-custom-cell-ts-a34fz5?file=Demo.tsx

**Comment by fhyami** [comment]
@komarovalexander Thank you so much, especially the quick response! 
I didn't initially understand how to access the data, but now I know to use props.rowData.[key]
